ntbs-service/Helpers/DrugResistanceHelper.cs

```diff
@@ -22,13 +22,15 @@ public static bool IsMbovis(DrugResistanceProfile profile)
             return string.Equals("M. bovis", profile.Species, StringComparison.InvariantCultureIgnoreCase);
         }
 
-        public static bool IsMBovisQuestionnaireComplete(bool? HasExposureToKnownMBovisCases,
-            bool? HasUnpasteurisedMilkConsumption, bool? HasOccupationMBovisExposure, bool? HasAnimalMBovisExposure)
+        public static bool IsMBovisQuestionnaireComplete(bool? hasExposureToKnownMBovisCasesValue,
+            bool? hasUnpasteurisedMilkConsumptionValue,
+            bool? hasOccupationMBovisExposureValue,
+            bool? hasAnimalMBovisExposureValue)
         {
-            return HasExposureToKnownMBovisCases != null
-                   && HasUnpasteurisedMilkConsumption != null
-                   && HasOccupationMBovisExposure != null
-                   && HasAnimalMBovisExposure != null;
+            return hasExposureToKnownMBovisCasesValue != null
+                   && hasUnpasteurisedMilkConsumptionValue != null
+                   && hasOccupationMBovisExposureValue != null
+                   && hasAnimalMBovisExposureValue != null;
```

Comment:
Sorry, I think you've misinterpreted what I meant here! I meant that instead of writing `HasExposureToKnownMBovisCases != null` you could just write `HasExposureToKnownMBovisCases.HasValue`. It's syntactic sugar, but the latter is preferred as it's slightly more readable.

Reply:
Ah sorry! I see what you meant! I've changed that over now